cfme/utils/appliance/__init__.py

```diff
@@ -2609,22 +2607,22 @@ def load_appliances(appliance_list, global_kwargs):
         List of :py:class:`IPAppliance`
     """
     result = []
-    for appliance_kwargs in appliance_list:
+    for num, appliance_kwargs in enumerate(appliance_list):
```

Comment:
**OPTIONAL** for clarity please use `idx` or `index`

Reply:
Done